src/data/nat/log.lean

```diff
@@ -128,6 +128,9 @@ begin
   simp [hn, this]
 end
 
+@[simp] lemma log_base_mul (b n : ℕ) (hb : 1 < b) (hn : 0 < n) : log b (b * n) = log b n + 1 :=
```

Comment:
```suggestion
@[simp] lemma log_base_mul (b n : ℕ) (hb : 1 < b) (hn : 0 < n) : log b (b * n) = 1 + log b n :=
```
feels more natural to me, since the order of the `*` matches the order of the `+`